mergify_engine/rules/__init__.py

```diff
@@ -241,34 +252,48 @@ async def create(
                         )
 
             # NOTE(sileht):
-            # In the summary, we display rules in four groups:
+            # In the summary, we display rules in five groups:
             # * rules where all attributes match -> matching_rules
             # * rules where only BASE_ATTRIBUTES match (filter out rule written for other branches) -> matching_rules
             # * rules that won't work due to a configuration issue detected at runtime (eg team doesn't exists) -> faulty_rules
             # * rules where only BASE_ATTRIBUTES don't match (mainly to hide rule written for other branches) -> ignored_rules
+            # * rules where only BASE_CHANGEABLE ATTRIBUTES don't match (they rarely change but are handled)-> not_applicable_base_changeable_attributes_rules
             categorized = False
+            evaluated_rule = typing.cast(T_EvaluatedRule, rule)
+            if rule_hidden_from_merge_queue and not rule.conditions.match:
+                # NOTE(sileht): Replace non-base attribute and non-base changeables attributes
+                # by true, if it still matches it's a potential rule otherwise hide it.
+                base_changeable_conditions = rule.conditions.copy()
+                for condition in base_changeable_conditions.walk():
+                    attr = condition.get_attribute_name()
+                    if attr not in self.BASE_CHANGEABLE_ATTRIBUTES:
+                        condition.update("number>0")
 
-            if hide_rule and not rule.conditions.match:
-                # NOTE(sileht): Replace non-base attribute by true, if it
-                # still matches it's a potential rule otherwise hide it.
                 base_conditions = rule.conditions.copy()
                 for condition in base_conditions.walk():
                     attr = condition.get_attribute_name()
                     if attr not in self.BASE_ATTRIBUTES:
                         condition.update("number>0")
 
+                await base_changeable_conditions(pulls)
                 await base_conditions(pulls)
 
-                if not base_conditions.match:
-                    self.ignored_rules.append(typing.cast(T_EvaluatedRule, rule))
+                if not base_changeable_conditions.match:
+                    self.not_applicable_base_changeable_attributes_rules.append(
+                        evaluated_rule
+                    )
+                    categorized = True
```

Comment:
We should not categorize them as we want them to be in the matching_rules set to have their actions running. This set is here only to the Summary UI to not show them if they don't pass.

Reply:
I don't understand, in this context if the rule is in the `not_applicable_base_changeable_attributes_rules` we are sure that it is not gonna pass due to the false `base_changeable_attributes` right ?